### PR TITLE
updated px image to 2.6.3

### PR DESCRIPTION
**Pull request by trierra**

### CODE REVIEW

File: charts/portworx/v1.1.1/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - portworx
+  - amd64
+home: https://portworx.com/
+maintainers:
+  - name: harsh-px
+    email: harsh@portworx.com
+  - name: trierra
+    email: oksana@portworx.com
+  - name: sharma-tapas
+    email: tapas@portworx.com
```
Comment (harsh-px) [inline charts/portworx/v1.1.1/Chart.yaml:21]: @trierra let's update the emails here

File: charts/portworx/v1.1.1/values.yaml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+csi: false                            # Enable CSI
+
+serviceAccount:
+  hook:
+    create: true
+    name:
+
+deploymentType: oci                   # accepts "oci" or "docker"
+imageType: none                       #
+imageVersion: 2.7.0                 # Version of the PX Image.
```
Comment (harsh-px) [inline charts/portworx/v1.1.1/values.yaml:44]: needs to be 2.6.3

**Comment by trierra** [comment]
done

**Comment by trierra** [comment]
NOTE: lint check is failing due to the presence of non-existent repo https://kubernetes-charts.storage.googleapis.com/ in other charts. There is nothing we can do in this PR to fix it

**Review by harsh-px — APPROVED** [review]

**Comment by harsh-px** [comment]
@PennyScissors can you help approve and merge this?

**Comment by PennyScissors** [comment]
Hello @trierra, could you please use Helm's chart testing tool to lint the chart? I'm seeing some problems after running it local as following. Thanks!

1. Add the rancher remote to your fork and fetch master as following
```
git remote add -ft master rancher-charts https://github.com/rancher/charts.git
```
2. Run ct lint:
```
docker run -v $(pwd):/rancher-charts -w /rancher-charts quay.io/helmpack/chart-testing:latest ct lint \
--remote rancher-charts \
--target-branch master \
--validate-maintainers=false \
--chart-dirs charts/portworx
```

Output:

```
 Charts to be processed:
------------------------------------------------------------------------------------------------------------------------
 portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1")
------------------------------------------------------------------------------------------------------------------------

Linting chart 'portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1")'
Checking chart 'portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1")' for a version bump...
Unable to find chart on master. New chart detected.
Validating /rancher-charts/charts/portworx/v1.1.1/Chart.yaml...
Validation failed!
Error validating data '/rancher-charts/charts/portworx/v1.1.1/Chart.yaml' with schema '/etc/ct/chart_schema.yaml'
        apiVersion: Required field missing
------------------------------------------------------------------------------------------------------------------------
 ✖︎ portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1") > Error waiting for process: exit status 1
------------------------------------------------------------------------------------------------------------------------
```

**Comment by PennyScissors** [comment]
On a side note, can I close this [PR](https://github.com/rancher/charts/pull/547)? Seems like it was waiting on the release of 2.6.0, but this PR updates to 2.6.3 so I'm assuming it might not be needed anymore and we just lost track of it.

**Comment by trierra** [comment]
Hello @PennyScissors , I'll run the Helm's chart testing tool on this PR. 
As for the [547](https://github.com/rancher/charts/pull/547), we would still like to get it merged since it contains some doc changes. 

Thanks!

**Comment by trierra** [comment]
I've run the provided commands and got the following output:
```
Charts to be processed:
------------------------------------------------------------------------------------------------------------------------
 portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1")
------------------------------------------------------------------------------------------------------------------------

Linting chart 'portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1")'
Checking chart 'portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1")' for a version bump...
Unable to find chart on master. New chart detected.
Validating /rancher-charts/charts/portworx/v1.1.1/Chart.yaml...
Validation failed!
Error validating data '/rancher-charts/charts/portworx/v1.1.1/Chart.yaml' with schema '/etc/ct/chart_schema.yaml'
        apiVersion: Required field missing
------------------------------------------------------------------------------------------------------------------------
 ✖︎ portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1") > Error waiting for process: exit status 1
------------------------------------------------------------------------------------------------------------------------
Error linting charts: Error processing charts
Error: Error linting charts: Error processing charts
```

**Comment by PennyScissors** [comment]
> I've run the provided commands and got the following output:
> 
> ```
> Charts to be processed:
> ------------------------------------------------------------------------------------------------------------------------
>  portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1")
> ------------------------------------------------------------------------------------------------------------------------
> 
> Linting chart 'portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1")'
> Checking chart 'portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1")' for a version bump...
> Unable to find chart on master. New chart detected.
> Validating /rancher-charts/charts/portworx/v1.1.1/Chart.yaml...
> Validation failed!
> Error validating data '/rancher-charts/charts/portworx/v1.1.1/Chart.yaml' with schema '/etc/ct/chart_schema.yaml'
>         apiVersion: Required field missing
> ------------------------------------------------------------------------------------------------------------------------
>  ✖︎ portworx => (version: "1.1.1", path: "charts/portworx/v1.1.1") > Error waiting for process: exit status 1
> ------------------------------------------------------------------------------------------------------------------------
> Error linting charts: Error processing charts
> Error: Error linting charts: Error processing charts
> ```

Based on that output, it looks like you are missing `apiVersion` in `Chart.yaml`

**Review by PennyScissors — APPROVED** [review]

**Pull request merged**